package.json

```diff
@@ -1,126 +1,35 @@
-  ],
-  "keywords": [
-    "async",
-    "react",
-    "redux",
-    "react-redux",
-    "hooks",
-    "ssr",
-    "fetch",
-    "data",
-    "query",
-    "promise"
-  ],
-  "license": "Apache-2.0",
   "scripts": {
-    "build": "babel src --out-dir lib",
+    "build": "yarn workspaces foreach run build",
     "lint": "eslint --ignore-path .gitignore --ext js,jsx,md .",
-    "prepublish": "npm run build",
-    "test": "npm run test:unit",
+    "test": "yarn test:unit",
     "test:unit": "jest",
     "test:git-history": "commitlint --from origin/beta --to HEAD",
-    "test:lockfile": "lockfile-lint -p package-lock.json -t npm -a npm -o https: -c -i",
-    "posttest": "npm run lint && npm run test:lockfile && npm run test:git-history"
-  },
-  "peerDependencies": {
-    "react": "16.x.x",
-    "react-dom": "16.x.x"
-  },
-  "optionalDependencies": {
-    "immutable": "^4.0.0-rc.12",
-    "react-redux": "7.x.x",
-    "redux": "4.x.x"
-  },
-  "dependencies": {
-    "@babel/runtime": "^7.11.2",
-    "create-shared-react-context": "^1.0.3",
-    "object-hash": "^2.0.3",
-    "prop-types": "^15.7.2"
+    "posttest": "yarn lint && yarn test:git-history"
   },
   "devDependencies": {
-    "@babel/cli": "^7.10.5",
-    "@babel/core": "^7.11.1",
-    "@babel/plugin-transform-runtime": "^7.11.0",
     "@babel/polyfill": "^7.10.4",
     "@commitlint/cli": "^9.1.1",
     "@commitlint/config-conventional": "^9.1.1",
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/commit-analyzer": "^8.0.1",
-    "@semantic-release/git": "^9.0.0",
-    "@semantic-release/github": "^7.0.7",
-    "@semantic-release/npm": "^7.0.5",
-    "@semantic-release/release-notes-generator": "^9.0.1",
-    "@testing-library/react": "^10.4.8",
+    "@testing-library/react": "^11.0.4",
     "amex-jest-preset-react": "^6.1.0",
-    "babel-preset-amex": "^3.4.0",
+    "babel-preset-amex": "^3.4.1",
     "core-js-compat": "3.6.5",
-    "eslint": "^7.6.0",
-    "eslint-config-amex": "^13.0.0",
+    "eslint": "^7.10.0",
+    "eslint-config-amex": "^13.1.0",
     "husky": "^4.2.5",
-    "immutable": "^4.0.0-rc.12",
-    "jest": "^26.4.0",
-    "lockfile-lint": "^4.3.7",
-    "react": "16.x.x",
-    "react-dom": "16.x.x",
-    "react-redux": "7.x.x",
-    "redux": "4.x.x",
-    "semantic-release": "^17.1.1"
-  },
-  "release": {
-    "branches": [
-      "+([0-9])?(.{+([0-9]),x}).x",
-      "main",
-      "next",
-      "next-major",
-      {
-        "name": "beta",
-        "prerelease": true
-      },
-      {
-        "name": "alpha",
-        "prerelease": true
-      }
-    ],
-    "plugins": [
-      "@semantic-release/commit-analyzer",
-      "@semantic-release/release-notes-generator",
-      "@semantic-release/changelog",
-      "@semantic-release/npm",
-      "@semantic-release/git",
-      "@semantic-release/github"
-    ]
+    "jest": "^26.5.2",
+    "lerna": "^3.22.1",
+    "semantic-release": "^17.2.1"
```

Comment:
With using `lerna`  do we still need semantic release?  Can you test out a release and see if it works. Otherwise we can look at reusing the process we have with one-app-cli or look at another tool

Reply:
removed it